.baseline/copyright/999_palantir.txt

```diff
@@ -0,0 +1 @@
+(c) Copyright ${today.year} Palantir Technologies Inc. All rights reserved.
```

Comment:
This should not be added

Reply:
Hmm this was generated by `baselineUpdateConfig`, will figure out why.

Reply:
This is also generated if you run `baselineUpdateConfig` on develop. I can remove it here, but I suspect that an excavator will automatically generate it in the future.

Reply:
The excavator deletes new copyright files added by the task, when running the update task we need to do that step manually